pod/perlintro.pod

```diff
@@ -172,8 +172,9 @@ Scalar values can be used in various ways:
  print "The animal is $animal\n";
  print "The square of $answer is ", $answer * $answer, "\n";
 
-There are a number of "magic" scalars with names that look like
-punctuation or line noise.  These special variables are used for all
+There are a number of "magic" scalars with names that use punctuation symbols,
```

Comment:
Making it even more positive:

`Perl defines a number of "magic" scalars with one-character names, often punctuation marks or digits.`

Reply:
I like your structure better than mine, and yours covers `$a` and `$b` too, which the original does not. I liked Neil's suggestion of *scalar variables* instead of *scalars*. You've done a lot more thinking about style than either Neil or me, so I wondered what your thoughts are on this. I wasn't 100% decided either way.

Also, having got a little further in that document and got to `%ENV` I realised that I'd count that as a variable to mention here. And in typing this comment I also realise that **it** isn't a scalar. So that sort of invalidates the previous sentence saying "scalars" or "scalar variables".

Reply:
Yes, also `@_`, `@ARGV`, `@INC`, `%INC`, and all of the typeglobs.

Reply:
All excellent points!

Perhaps something like `Perl defines a number of "magic" variables with short names, often single punctuation marks or digits`? That doesn't discount our friends like %ENV.